src/main/kotlin/de/timo_reymann/mjml_support/tagprovider/BuiltInMjmlTagInformationProvider.kt

```diff
@@ -1235,7 +1312,11 @@ class BuiltInMjmlTagInformationProvider : MjmlTagInformationProvider() {
                         "left/right/center/justify",
                         "left"
                     ),
-                    ATTRIBUTE_CONTAINER_BACKGROUND_COLOR,
+                    MjmlAttributeInformation(
+                        "container-background-color",
+                        MjmlAttributeType.COLOR,
+                        "inner element background color"
+                    ),
```

Comment:
chore: Please refactor the container-background-color attributes into a constant like you did for `ATTRIBUTE_BACKGROUND_POSITION_*`